### PR TITLE
fix(home-page): Firefox Related CSS errors

### PR DESCRIPTION
**Pull request by new5558**
**Always tag `🚀 READY FOR REVIEW` once your PR is Ready to be one of us :)**

We value your endeavors to enhance the Hacktoberfest Museum better!

## If (this.PR.category === "The new entry 💫")

**Please tag `The new entry 💫` to the PR in order to categorize yours as the star being recognized by the Museum.**

## Else if (this.PR.category === "Enhancement 🛠")

**Please tag `Enhancement 🛠` to the PR as you're developing the Museum more wonderfully!**

## Finally:

> 🪄 **DON'T FORGET TO CLAIM YOUR TICKET!** via [Eventpop application form.](https://www.eventpop.me/events/13854/application_forms/845/applicants/new)

### CODE REVIEW

**Comment by new5558** [comment]
Fixes:
- Firefox link tag is not working per https://github.com/nolimits4web/atropos/issues/13#issuecomment-1117839950
- Firefox MacOS scrollbar not hidden https://stackoverflow.com/questions/20997183/how-to-hide-scrollbar-in-firefox
image

File: src/pages/index.astro [hunk 1]
```diff
@@ -48,3 +48,10 @@ const defaultEntries = entries.map((entry) => entry.default);
     </div>
   </body>
 </html>
+
```
Comment (Gusb3ll) [inline src/pages/index.astro]:
```suggestion

```

Extra line here kub :eyes:

**Review by Gusb3ll — CHANGES_REQUESTED** [review]
Please also run `pnpm prettier:write` to fix any formatting error krub~ :pray:

File: src/pages/index.astro (hunk 1 above)
Comment (new5558) [reply src/pages/index.astro]: Should we do this as git commit hooks?

Comment (Gusb3ll) [reply src/pages/index.astro]: Not sure if we integrate tools such as husky will impact any dx on the new comer side, but will take it to consideration kub :pray:

**Review by Gusb3ll — APPROVED** [review]

**Review by leomotors — APPROVED** [review]

**Comment by Gusb3ll** [comment]
@Leomotors LMAO

**Pull request merged**